docs/guides/using-vpc-bastion-ssh.md

```diff
@@ -0,0 +1,135 @@
+---
+page_title: "Using Scaleway SSH Bastion"
+description: |-
+  Using Scaleway SSH Bastion config.
+---
+
+# How to use Scaleway VPC SSH Bastion config
+
+In this guide you'll learn how to deploy Scaleway SSH bastion to your Scaleway Private Network using the Scaleway Terraform provider.
+After Bastion is deployed, you can connect (SSH) to virtual machines in the virtual network via Bastion using the private IP address of the VM.
+When you connect to a VM, it doesn't need a public IP address, client software, agent, or a special configuration.
+
+## Prerequisites
+
+*	A Virtual Machine in a VPC PrivateNetwork. Check our example below.
+
+1. When you deploy Bastion, the values are pulled from the Network in which your VM resides.
+   1. This VM doesn't become a part of the Bastion deployment itself, but you do connect to it later in the exercise.
+
+2. If you don't have any VMs connected to the Private Network, use the `scaleway_instance_private_nic` or the attribute `private_network` on `scaleway_instance_server` to connect.
+
+3. If your VMs are attached to any `scaleway_instance_ip`. You should detach them.
+
+  **Note**: You should keep your VMs and Private Network on the same zone. e.g. `fr-par-1`
+
+```hcl
+# NUMBER OF VIRTUAL MACHINES
+variable "machine_count" {
+  description = "Number of virtual machines in private network"
+  default = 3
+}
+
+# SCALEWAY VPC PRIVATE NETWORK
+resource scaleway_vpc_private_network "pn" {
+  name = "myprivatenetwork"
+}
+
+# SCALEWAY VPC VIRTUAL MACHINES
+resource scaleway_instance_server "servers" {
+  count	= var.machine_count
+  name 	= "machine${count.index}"
+  image = "ubuntu_focal"
+  type  = "DEV1-S"
+}
+
+# SCALEWAY INSTANCES PRIVATE NETWORK CONNECTION
+resource scaleway_instance_private_nic "nic" {
+  count              = length(scaleway_instance_server.servers)
+  private_network_id = scaleway_vpc_private_network.pn.id
+  server_id          = scaleway_instance_server.servers[count.index].id
+}
+```
+
+## Reserve a public gateway IP
+
+You should reserve your public IP, allowing it to reach the public internet, as well as forward (masquerade) traffic from member instances of attached Private Networks.
+
+This IP is a static IPv4 address designed for dynamic cloud computing.
+
+```hcl
+# SCALEWAY PUBLIC GATEWAY IP
+resource scaleway_vpc_public_gateway_ip "pgw_ip" {
+}
+```
+
+## Set up your Public Gateway
+
+Public Gateways sit at the border of Private Networks and allow you to enable the bastion.
+You can also choose your port of preference on `bastion_port` option.
+
+You can check the available types of gateways supported with our CLI.
+
+```shell
+scw vpc-gw gateway-type list
+```
+
+Example:
+
+```hcl
+resource scaleway_vpc_public_gateway "pgw" {
+  type = "VPC-GW-S"
+  bastion_enabled = true
+  ip_id = scaleway_vpc_public_gateway_ip.pgw_ip.id
+}
+```
+
+## Configure your DHCP on your subnet
+
+To communicate over the network we require the IP address.
```

Comment:
Who is the we? Could you precise that it is the instance and they require a an IPv4 address that would be private.